Framework/Algorithms/test/GetAllEiTest.h

```diff
@@ -521,15 +522,15 @@ class GetAllEiTest : public CxxTest::TestSuite {
     double t_chop(delay + inital_chop_phase / chopSpeed);
     double Period =
         (0.5 * 1.e+6) / chopSpeed; // 0.5 because some choppers open twice.
-    auto &x = ws->dataX(0);
+    auto &x = ws->mutableX(0);
     for (size_t i = 0; i < x.size(); i++) {
       x[i] = 5 + double(i) * 10;
     }
     // signal at first monitor
     double t1 = t_chop * l_mon1 / l_chop;
     double t2 = (t_chop + Period) * l_mon1 / l_chop;
     {
-      auto &y = ws->dataY(0);
+      auto &y = ws->mutableY(0);
       for (size_t i = 0; i < y.size(); i++) {
         double t = 0.5 * (x[i] + x[i + 1]);
```

Comment:
Again `Histogram` can do this for you, just use `points()` instead of `x()` or `mutableX()` (but pay attention: `points()` returns by value!).